src/components/ChatLog.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import ChatEntry from './ChatEntry'
+import './ChatEntry.css'
+import './ChatLog.css'
+
+
+function ChatLog(props) {
+    const [timer, setTimer] = React.useState(0)
+    React.useEffect(()=> {
+      const interval = setInterval(() => {
+        setTimer(timer + 1);
+    }, 1000);
+      return () => clearInterval(interval);
+    }, [timer]);
+
+    const chatComponent = props.entries.map((chat, index) => {
+    return timer>=index*3 ? ( <section key={chat.id}>
+    <ChatEntry updateLike = {props.updateLike} sender={chat.sender} body={chat.body} timeStamp={chat.timeStamp} liked={chat.liked} id={chat.id} ></ChatEntry>
+    </section>) : null
+  })
+    const chatComponent1 = props.entries.map((chat, index) => {
+    return ( <section key={chat.id}>
+    <ChatEntry updateLike = {props.updateLike} sender={chat.sender} body={chat.body} timeStamp={chat.timeStamp} liked={chat.liked} id={chat.id} ></ChatEntry>
+    </section>) 
+  })
+
+  return (
+    <section> 
+        {chatComponent1}
+    </section>
+    
+  )
+}
```

Comment:
Seems like this component is missing. `PropTypes`, what props do you think would be necessary to validate here? `PropTypes` helps us mitigate mistakes by checking to see if the props that were passed down to a component are present and look how we expect.